### PR TITLE
Fixes #85 - posixify now works with JSON downloads

### LINKED ISSUE #85: Write a unit test for pulling data from JSON
Need to ensure that `posixify()` worked both with csv and json files. Right now we're only testing CSV files.

### PR DESCRIPTION
**Pull request by nicklucius**
Fixes #85

### CODE REVIEW

**Comment by tomschenkjr** [comment]
I re-built the pull request in AppVeyor and it now passes.

@geneorama - can you do a review of this pull request? Click "[Add Your Review](https://help.github.com/articles/about-pull-request-reviews/)" to make any notes. It's for the v1.7.1 milestone that we'll submit on 10/31.

File: tests/testthat/test-all.R [hunk 1]
```diff
@@ -3,18 +3,29 @@ library(RSocrata)
 library(httr)
 library(jsonlite)
 library(mime)
+library(plyr)
```
Comment (geneorama) [inline tests/testthat/test-all.R]: `plyr` is not needed for the tests.  If it's a package dependency it belongs in the appropriate places in the NAMESPACE and DESCRIPTION files.

File: R/RSocrata.R [hunk 2]
```diff
@@ -109,20 +109,26 @@ posixify <- function(x) {
...
-	nMatchesShort <- grep(pattern = patternShort, x)
-	nMatchesLong <- grep(pattern = patternLong, x)
-	## Parse as the most likely calendar date format. Ties go to short format.
-	if(length(nMatchesLong) > length(nMatchesShort)){
-	  as.POSIXct(strptime(x, format="%m/%d/%Y %I:%M:%S %p")) # long date-time format
-	}	else {
-	  as.POSIXct(strptime(x, format="%m/%d/%Y")) # short date format
+	patternJSON <- paste0("^[[:digit:]]{4}-[[:digit:]]{2}-[[:digit:]]{2}T",
+	                      "[[:digit:]]{2}:[[:digit:]]{2}:[[:digit:]]{2}.[[:digit:]]{3}","$")
+		## Find number of matches with grep
```
Comment (geneorama) [inline R/RSocrata.R]: indenting (mentioned in commit message)

File: tests/testthat/test-all.R (hunk 1 above)
Comment (nicklucius) [inline tests/testthat/test-all.R]: `plyr` is in NAMESPACE and DESCRIPTION so looks good to me.

**Comment by nicklucius** [comment]
Thanks, Gene. The changes look good to me.

**Comment by geneorama** [comment]
I was going to do a "squash merge" but I don't want to lose authorship.  @nicklucius would you mind doing that?  In the commit message please take out my "removing plyr" and "adding plyr" messages.

**Pull request merged**